build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+}
+
+group = 'com.cag.cagbackendapi'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+	implementation "org.apache.tomcat.embed:tomcat-embed-core:9.0.35"
+	implementation "org.springframework.boot:spring-boot-starter-data-jpa"
+	implementation "com.nimbusds:nimbus-jose-jwt:8.2.1"
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation "org.springframework.retry:spring-retry:1.2.4.RELEASE"
+	implementation "org.springframework.boot:spring-boot-starter-aop:1.5.22.RELEASE"
+	implementation "org.springframework.boot:spring-boot-starter-actuator"
+	implementation "org.springframework.boot:spring-boot-starter-data-rest"
+	implementation 'junit:junit:4.12'
+	implementation "org.springframework:spring-web:5.2.3.RELEASE"
+	implementation "org.dom4j:dom4j:2.1.3"
+	implementation "com.fasterxml.jackson.module:jackson-module-kotlin:2.10.0"
+	implementation "org.jetbrains.kotlin:kotlin-reflect:1.3.50"
+	implementation "com.google.code.gson:gson:2.8.5"
+	//implementation "org.flywaydb:flyway-core"
+	implementation "io.micrometer:micrometer-registry-prometheus"
+	implementation "com.fasterxml.jackson.core:jackson-core:2.10.0"
+	implementation "org.springdoc:springdoc-openapi-ui:1.2.24"
+	implementation "io.swagger:swagger-annotations:1.5.0"
+	implementation "com.fasterxml.jackson.core:jackson-databind:2.10.0"
+
+	runtimeOnly "org.postgresql:postgresql:42.2.13"
```

Comment:
Can it be assumed that this only happens at run time? And what do the numbers mean?

Reply:
The numbers denote the version number of the postgresql library. For details on the difference between implementation and runtime and what not you can see this link: https://stackoverflow.com/questions/47365119/gradle-dependency-configuration-implementation-vs-api-vs-runtimeonly-vs-compil 

I don't really focus too much on those details because once the library is working correctly you normally don't have to worry about this. It's good knowledge I'm sure!